R/Factbase.R

```diff
@@ -342,6 +345,20 @@ value_type_for_vector <- function(v, column_name) {
         stop(paste('Cannot work out which data type to use for column', column_name, 'containing a', typeof(v), 'vector.  Only Date, POSIXt, text or real are accepted'))
 }
 
+ensureDefinitionHyperlinkOwnerSupplied <- function(definition, hyperlink, owner) {
+    if (is.null(definition))
+        stop("argument \"definition\" is missing")
+    if (!is.character(definition) || length(definition) != 1)
+        stop("argument \"definition\" must be a character vector of length 1, or null")
```

Comment:
Stating `"definition"` can be be `NULL` contradicts the lines above. 

A related aside, `is.character(NULL)` returns `FALSE` so can be removed unless you wanted visibility of a missing arg?

Reply:
Good spot.